notebooks/image_models/solutions/deepconv_gan.ipynb

```diff
@@ -50,33 +50,6 @@
     "## Import TensorFlow and other libraries"
```

Comment:
<p>Move this install into the top level <code>requirements.txt</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>It seems imageio is preinstalled and this line is not needed.</p><p>I will just remove it so far. Thanks!</p>